packages/@aws-cdk/aws-iam/lib/policy-document.ts

```diff
@@ -1,4 +1,5 @@
 import cdk = require('@aws-cdk/cdk');
+import { Region } from '@aws-cdk/region-settings';
```

Comment:
Don't care for the name "settings" much. They're not actually in our control (we can't "set" anything).

I'd prefer `aws-regions` but I see how we don't want to do that to avoid conflicting with service packages.

How about:

```
regions-aws (hah I can just revert those components)
regions
region-info
```

Reply:
Kinda like `region-info`. That sounds descriptive enough, and I also didn't like settings, but couldn't come up with a better name ¯\\\_(シ)\_/¯ .

Reply:
+1 "regions"